src/buildercore/terraform.py

```diff
@@ -530,13 +532,18 @@ def add_table(table_id, table_options):
         schema = table_options['schema']
         stackname = context['stackname']
         fqrn = "%s_%s" % (table_options['dataset_id'], table_id) # 'fully qualified resource name'
+        github_token = False
 
         if schema.startswith('https://'):
             # remote schema, add a 'http' provider and have terraform pull it down for us
             # https://www.terraform.io/docs/providers/http/data_source.html
             tf_file['data'][DATA_TYPE_HTTP][fqrn] = {'url': schema}
             schema_ref = '${data.http.%s.body}' % fqrn
-
+            if re.match('^https://raw\.githubusercontent\.com/', schema):
```

Comment:
(might as well use `startswith`?)

Reply:
Might also be missing the `r` prefix if this is to use a regular expression. Otherwise '\.' would ask it to (un)escape '.' as a string, rather than a regular expression.

Reply:
It kinda works because there is no meaning for `\.` like there is for `\n`, but yes I'll make it robust